Neos.Cache/Classes/Backend/FileBackend.php

```diff
@@ -281,7 +281,11 @@ public function flush(): void
     {
         Files::emptyDirectoryRecursively($this->cacheDirectory);
         if ($this->frozen === true) {
-            @unlink($this->cacheDirectory . 'FrozenCache.data');
+            try {
+                @unlink($this->cacheDirectory . 'FrozenCache.data');
+            } catch (\Throwable $e) {
+                // Double-shrug catching
```

Comment:
Yeah, you might wonder about this bad practice. PHP 8 apparently now throws for unlink even with shutup operator, but we really don't care at those places. It's also the only way to handle this race-condition free.

Reply:
Is "double-shrug" a catchphrase? I've never heard of it and maybe we should just make your Github comment an inline comment instead :)

Reply:
Not really, just read it in a blog post about PHP 8 here https://php.watch/versions/8.0#fatal-error-suppression and found it somewhat amusing
![image](https://user-images.githubusercontent.com/4259532/111123572-ca7d3780-856f-11eb-8d95-f3f9991970ef.png)

But 👍 for making this more helpful

Reply:
```suggestion
                // PHP 8 apparently throws for unlink even with shutup operator, but we really don't care at this place. It's also the only way to handle this race-condition free.
```